versioned_docs/version-2.x/advanced/server/add-http-endpoint.mdx

```diff
@@ -181,3 +181,42 @@ export default {
   },
 };
 ```
+
+## Extend the HTTP server with `onServerInit`
+
+<SinceVersion tag="2.30" />
+
+The `onServerInit` hook is a server module extension point introduced in
+Front-Commerce 2.30. It allows you to extend the Express HTTP server or perform
+additional tasks during server initialization.
```

Comment:
nitpick: the `SinceVersion` already explains that it's introduced in `Front-Commerce 2.30` I don't think it's necessary to repeat it.

 ```suggestion
The `onServerInit` hook allows you to extend the Express HTTP server or perform
additional tasks during server initialization.
```

Reply:
Updated with your suggestion